gradle/scala.gradle

```diff
@@ -49,12 +49,18 @@ test {
 sourceSets {
     main {
         scala {
-            srcDir "src/main/scala_$scalaBaseVersion"
+            srcDirs += ["src/main/scala_$scalaBaseVersion", "src/main/java"]
+        }
+        java {
+            srcDirs = []
         }
     }
```

Comment:
@mmalohlava yes but that's because it has no dependencies on Scala code. `JavaH2OContext` for example does (`SupportedRDD$`) and needs to be in the same srcDir as Scala sources.

Also the `java { srcDirs = [] }` part is necessary because otherwise it will try to compile Java classes twice (once for Scala compilation since I added it to Scala's `srcDirs` and once for Java source dirs -> this will fail because no Scala classes in the Java `srcDirs`).